### ISSUE TITLE
Status message are not announced by screen readers

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1624]
When attempting to delete your account or perform other actions on the page, status messages such as ‘invalid username or password’ appear on the screen. However, none of these status messages are announced by screen readers, preventing screen reader users from
understanding the outcomes of their actions. Without audible feedback, these users are left unaware of critical information, such as errors or confirmations.
